fsspec/implementations/tests/local/local_fixtures.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from fsspec.implementations.local import LocalFileSystem
+from fsspec.tests.abstract import AbstractFixtures
+
+
+class LocalFixtures(AbstractFixtures):
+    @staticmethod
+    @pytest.fixture
+    def fs():
+        return LocalFileSystem(auto_mkdir=True)
+
+    @staticmethod
+    @pytest.fixture
+    def fs_path(tmpdir):
+        return str(tmpdir)
```

Comment:
You probably want this posix-ised, since we explicitly use "/" elsewhere in the tests, and windows will only accept posix paths if they are wholly posix.